pages/common/xsv.md

```diff
@@ -0,0 +1,27 @@
+# xsv
+
+> A fast CSV command line toolkit written in Rust.
+
+- Inspect the headers of a file:
+
+`xsv headers {{path/to/worldcities.csv}}`
```

Comment:
worldcities sounds odd. We generally use path/to/file.csv. Can we use that everywhere ?